src/api/handler/handler.js

```diff
@@ -1,17 +1,25 @@
-import {axiosNasaClient} from "../../clients/index.js";
+import {nasaAsteroidService} from "../../services/index.js";
 import {getPreviousPeriod, parseDate} from "./dateUtils.js";
 
+const parseQueryParams = (query) => {
+    return {
+        date: query.date,
+        countOnly: Boolean(query.count),
+        dangerousOnly: Boolean(query["were-dangerous-meteors"]),
+    }
+}
+
 export const handler = async (req, res) => {
-    const dateQuery = req.query.date;
-    const {errorMessage, date} = parseDate(dateQuery);
+    const {date, countOnly, dangerousOnly} = parseQueryParams(req.query);
+    const {errorMessage} = parseDate(date);
     if (errorMessage) {
         res.status(400).send(errorMessage);
         return;
     }
 
     try {
         const period = getPreviousPeriod(date);
-        const response = await axiosNasaClient.getAsteroidsCountByPeriod(period);
+        const response = await nasaAsteroidService.getAsteroidsCountByPeriod(period, countOnly, dangerousOnly);
         res.set("Content-Type", "application/json")
         res.send(JSON.stringify(response, null, 2));
```

Comment:
inconsistent entry
first you indicate that the content is in json format and then convert the json to a string
```js
const response = await nasaAsteroidService.getAsteroidsCountByPeriod(period, countOnly, dangerousOnly);
res.json(response);
```

Reply:
Fixed!